### PR TITLE
let the path /apisix/admin support the CORS

### PR DESCRIPTION
**Pull request by Miss-you**
…ORS.

NOTE: Please read the Contributing.md guidelines before submitting your patch:

https://github.com/apache/incubator-apisix/blob/master/Contributing.md#how-to-add-a-new-feature-or-change-an-existing-one

### Summary

fix bin/apisix, let the API /apisix/admin support the CORS

### CODE REVIEW

**Comment by lilien1010** [comment]
compare to make `/apisix/admin` CORS enabled in a  hardcode-way,

I think we can move those code into apisix.http_admin,which I think It will make things more elegant

**Comment by lilien1010** [comment]
here is my suggestion for the admin api CORS feature.

1、set a enable_cors switch in config.yaml  just like  admin_port
2、move those CORS hardcode into function apisix.http_admin()

**Comment by Miss-you** [comment]
I think it is necessary to add a enable_admin_cors switch.
However, moving the code from nginx.conf to http_admin is still hard-coded and does not solve the problem you said, right? Should we add another admin API CORS configuration?

**Comment by moonming** [comment]
@Miss-you Travis CI is broken.

File: lua/apisix.lua [hunk 1]
```diff
@@ -446,6 +446,29 @@ function _M.http_balancer_phase()
     load_balancer(api_ctx.matched_route, api_ctx)
 end
 
+local function cors_admin()
+    local local_conf = core.config.local_conf()
+    if local_conf.apisix or not local_conf.apisix.enable_admin_cors then
```
Comment (moonming) [inline lua/apisix.lua]: will be always `true`, you should add test case for this PR.

Comment (Miss-you) [reply lua/apisix.lua]: I found this problem during my self-test, but forgot to submit the code

Comment (Miss-you) [reply lua/apisix.lua]: _M.init_worker() in lua/apisix/admin/init.lua also has the same problem.

Comment (moonming) [reply lua/apisix.lua]: > _M.init_worker() in lua/apisix/admin/init.lua also has the same problem.

welcome create another PR to fix it.

Comment (Miss-you) [reply lua/apisix.lua]: ok

**Comment by moonming** [comment]
merged, thx @Miss-you

**Pull request merged**